packages/client/wallets/aa/src/types/Config.ts

```diff
@@ -6,16 +6,15 @@ export type SmartWalletSDKInitParams = {
     clientApiKey: string;
 };
 
-export type UserIdentifierParams = {
-    email?: string;
-    userId?: string;
-    phoneNumber?: string;
+type WhitelabelUserParams = {
+    /**
+     * A unique identifier for the user. This must match the value of the identifier within the JWT
+     * that is specified in the project settings (typically `sub`).
+     */
+    id: string;
+    jwt: string;
 };
-
-export type UserIdentifier =
-    | { type: "whiteLabel"; userId: string }
-    | { type: "email"; email: string }
-    | { type: "phoneNumber"; phoneNumber: string };
+export type UserParams = WhitelabelUserParams;
```

Comment:
thinking through future-proofness of this
should we keep a label (e.g. `{ type: "whiteLabel";`) on each scenario so that we can still easily `switch` on it when we need to handle?

Reply:
Going to land this PR and circle back on this before release, I think there's a way we can both:
- Have a good default passkey name from the user
- Drop that `id` property, only passing in the JWT in `UserParams`

So devs could call `SmartWalletSDK.getOrCreateWallet({jwt}, "base")`, and we could derive a bunch of details from the JWT:
- The id, like you mentioned
- Whether or not it should be grey label based on the issuer of the token.

In which case we won't need the label anymore